msal/samples/desktop/SampleApp/MsalAuthHelper.cs

```diff
@@ -98,5 +112,32 @@ public async Task<string> GetTokenForCurrentAccountAsync(IEnumerable<string> sco
             return null;
         }
 
+        internal async Task<string> GetTokenWithIWAAsync(string[] scopes, string user)
+        {
+            AuthenticationResult result = null;
+            Exception exception = null;
+
+            try
+            {
+                result = await Application.AcquireTokenByIntegratedWindowsAuthAsync(scopes, user).ConfigureAwait(false);
+                return result.AccessToken;
+            }
+            catch (MsalServiceException ex)
```

Comment:
You can just catch all exception, like at line 130 and then show the message box, like at line 137.